cmd/admin-handlers.go

```diff
@@ -2943,7 +2943,7 @@ function main() {
 	MINIO_OPTS=$(grep "Server command line args" <./inspect-input.txt | sed "s/Server command line args: //g" | sed -r "s#https:\/\/#\.\/#g")
 
 	# Start MinIO instance using the options
-	START_CMD="CI=on MINIO_ROOT_USER=minio MINIO_ROOT_PASSWORD=minio123 minio server ${MINIO_OPTS} &"
```

Comment:
We should also turn-off auto healing if drives with inspection script by default

Reply:
Ack. Will change.